<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="20" ignored="0" total="20" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="2410837" started-at="2019-11-19T11:17:02Z" finished-at="2019-11-19T11:57:13Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="2410837" started-at="2019-11-19T11:17:02Z" finished-at="2019-11-19T11:57:13Z">
      <class name="core.tests.Package">
        <test-method status="PASS" signature="runOncePerSuite()[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerSuite" is-config="true" duration-ms="207" started-at="2019-11-19T16:47:02Z" finished-at="2019-11-19T16:47:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerSuite -->
        <test-method status="PASS" signature="runOncePerContext(org.testng.ITestContext)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerContext" is-config="true" duration-ms="18" started-at="2019-11-19T16:47:02Z" finished-at="2019-11-19T16:47:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerContext -->
        <test-method status="PASS" signature="runOncePerClass(org.testng.ITestContext)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerClass" is-config="true" duration-ms="1" started-at="2019-11-19T16:47:02Z" finished-at="2019-11-19T16:47:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerClass -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="23597" started-at="2019-11-19T16:47:02Z" finished-at="2019-11-19T16:47:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreBellandurSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreBellandurSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreBellandurSignaturePackage" duration-ms="100325" started-at="2019-11-19T16:47:25Z" finished-at="2019-11-19T16:49:06Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreBellandurSignaturePckg(HomePage.java:516)at core.tests.Package.tc_Packge_BloreBellandurSignaturePackage(Package.java:90)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreBellandurSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1420" started-at="2019-11-19T16:49:06Z" finished-at="2019-11-19T16:49:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreBellandurSignaturePackage status=FAILURE method=Package.tc_Packge_BloreBellandurSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="23247" started-at="2019-11-19T16:49:07Z" finished-at="2019-11-19T16:49:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreJayanagarLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreJayanagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreJayanagarLuxuryPackage" duration-ms="98960" started-at="2019-11-19T16:49:31Z" finished-at="2019-11-19T16:51:09Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreJayanagarLuxuryPckg(HomePage.java:421)at core.tests.Package.tc_Packge_BloreJayanagarLuxuryPackage(Package.java:50)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreJayanagarLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1428" started-at="2019-11-19T16:51:09Z" finished-at="2019-11-19T16:51:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreJayanagarLuxuryPackage status=FAILURE method=Package.tc_Packge_BloreJayanagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22263" started-at="2019-11-19T16:51:11Z" finished-at="2019-11-19T16:51:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreJayanagarSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreJayanagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreJayanagarSignaturePackage" duration-ms="97983" started-at="2019-11-19T16:51:33Z" finished-at="2019-11-19T16:53:11Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreJayanagarSignaturePckg(HomePage.java:402)at core.tests.Package.tc_Packge_BloreJayanagarSignaturePackage(Package.java:42)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreJayanagarSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1365" started-at="2019-11-19T16:53:11Z" finished-at="2019-11-19T16:53:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreJayanagarSignaturePackage status=FAILURE method=Package.tc_Packge_BloreJayanagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22413" started-at="2019-11-19T16:53:13Z" finished-at="2019-11-19T16:53:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreMalleshwaramLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreMalleshwaramLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreMalleshwaramLuxuryPackage" duration-ms="97922" started-at="2019-11-19T16:53:35Z" finished-at="2019-11-19T16:55:13Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreMalleshwaramLuxuryPckg(HomePage.java:383)at core.tests.Package.tc_Packge_BloreMalleshwaramLuxuryPackage(Package.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreMalleshwaramLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1292" started-at="2019-11-19T16:55:13Z" finished-at="2019-11-19T16:55:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreMalleshwaramLuxuryPackage status=FAILURE method=Package.tc_Packge_BloreMalleshwaramLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22232" started-at="2019-11-19T16:55:14Z" finished-at="2019-11-19T16:55:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreMalleshwaramSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreMalleshwaramSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreMalleshwaramSignaturePackage" duration-ms="97563" started-at="2019-11-19T16:55:36Z" finished-at="2019-11-19T16:57:14Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreMalleshwaramSignaturePckg(HomePage.java:364)at core.tests.Package.tc_Packge_BloreMalleshwaramSignaturePackage(Package.java:26)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreMalleshwaramSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1321" started-at="2019-11-19T16:57:14Z" finished-at="2019-11-19T16:57:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreMalleshwaramSignaturePackage status=FAILURE method=Package.tc_Packge_BloreMalleshwaramSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22262" started-at="2019-11-19T16:57:15Z" finished-at="2019-11-19T16:57:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreOARLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreOARLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreOARLuxuryPackage" duration-ms="97625" started-at="2019-11-19T16:57:38Z" finished-at="2019-11-19T16:59:15Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreOARLuxuryPckg(HomePage.java:497)at core.tests.Package.tc_Packge_BloreOARLuxuryPackage(Package.java:82)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreOARLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1336" started-at="2019-11-19T16:59:15Z" finished-at="2019-11-19T16:59:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreOARLuxuryPackage status=FAILURE method=Package.tc_Packge_BloreOARLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22217" started-at="2019-11-19T16:59:16Z" finished-at="2019-11-19T16:59:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreOARSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreOARSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreOARSignaturePackage" duration-ms="95541" started-at="2019-11-19T16:59:39Z" finished-at="2019-11-19T17:01:14Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreOARSignaturePckg(HomePage.java:478)at core.tests.Package.tc_Packge_BloreOARSignaturePackage(Package.java:74)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreOARSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1346" started-at="2019-11-19T17:01:14Z" finished-at="2019-11-19T17:01:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreOARSignaturePackage status=FAILURE method=Package.tc_Packge_BloreOARSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22333" started-at="2019-11-19T17:01:16Z" finished-at="2019-11-19T17:01:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreWhitefieldLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreWhitefieldLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreWhitefieldLuxuryPackage" duration-ms="95820" started-at="2019-11-19T17:01:38Z" finished-at="2019-11-19T17:03:14Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreWhitefieldLuxuryPckg(HomePage.java:459)at core.tests.Package.tc_Packge_BloreWhitefieldLuxuryPackage(Package.java:66)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreWhitefieldLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1330" started-at="2019-11-19T17:03:14Z" finished-at="2019-11-19T17:03:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreWhitefieldLuxuryPackage status=FAILURE method=Package.tc_Packge_BloreWhitefieldLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="21957" started-at="2019-11-19T17:03:15Z" finished-at="2019-11-19T17:03:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_BloreWhitefieldSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_BloreWhitefieldSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_BloreWhitefieldSignaturePackage" duration-ms="98431" started-at="2019-11-19T17:03:37Z" finished-at="2019-11-19T17:05:15Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.bloreWhitefieldSignaturePckg(HomePage.java:440)at core.tests.Package.tc_Packge_BloreWhitefieldSignaturePackage(Package.java:58)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_BloreWhitefieldSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1328" started-at="2019-11-19T17:05:15Z" finished-at="2019-11-19T17:05:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_BloreWhitefieldSignaturePackage status=FAILURE method=Package.tc_Packge_BloreWhitefieldSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="23097" started-at="2019-11-19T17:05:17Z" finished-at="2019-11-19T17:05:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_GurgaonSector47SignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_GurgaonSector47SignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_GurgaonSector47SignaturePackage" duration-ms="96723" started-at="2019-11-19T17:05:40Z" finished-at="2019-11-19T17:07:17Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.gurgaonSector47SignaturePckg(HomePage.java:687)at core.tests.Package.tc_Packge_GurgaonSector47SignaturePackage(Package.java:162)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_GurgaonSector47SignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1356" started-at="2019-11-19T17:07:17Z" finished-at="2019-11-19T17:07:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_GurgaonSector47SignaturePackage status=FAILURE method=Package.tc_Packge_GurgaonSector47SignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22719" started-at="2019-11-19T17:07:18Z" finished-at="2019-11-19T17:07:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_HRBRLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_HRBRLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_HRBRLuxuryPackage" duration-ms="83278" started-at="2019-11-19T17:07:41Z" finished-at="2019-11-19T17:09:04Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.hrbrLuxuryPckg(HomePage.java:345)at core.tests.Package.tc_Packge_HRBRLuxuryPackage(Package.java:18)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_HRBRLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1459" started-at="2019-11-19T17:09:04Z" finished-at="2019-11-19T17:09:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_HRBRLuxuryPackage status=FAILURE method=Package.tc_Packge_HRBRLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22657" started-at="2019-11-19T17:09:05Z" finished-at="2019-11-19T17:09:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_HRBRSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_HRBRSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_HRBRSignaturePackage" duration-ms="81934" started-at="2019-11-19T17:09:28Z" finished-at="2019-11-19T17:10:50Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.hrbrSignaturePckg(HomePage.java:332)at core.tests.Package.tc_Packge_HRBRSignaturePackage(Package.java:10)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_HRBRSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1367" started-at="2019-11-19T17:10:50Z" finished-at="2019-11-19T17:10:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_HRBRSignaturePackage status=FAILURE method=Package.tc_Packge_HRBRSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22389" started-at="2019-11-19T17:10:51Z" finished-at="2019-11-19T17:11:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_MumbaiMaladLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_MumbaiMaladLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_MumbaiMaladLuxuryPackage" duration-ms="97931" started-at="2019-11-19T17:11:14Z" finished-at="2019-11-19T17:12:52Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.mumbaiMaladLuxuryPckg(HomePage.java:554)at core.tests.Package.tc_Packge_MumbaiMaladLuxuryPackage(Package.java:106)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_MumbaiMaladLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1377" started-at="2019-11-19T17:12:52Z" finished-at="2019-11-19T17:12:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_MumbaiMaladLuxuryPackage status=FAILURE method=Package.tc_Packge_MumbaiMaladLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="23283" started-at="2019-11-19T17:12:53Z" finished-at="2019-11-19T17:13:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_MumbaiMaladSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_MumbaiMaladSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_MumbaiMaladSignaturePackage" duration-ms="98955" started-at="2019-11-19T17:13:16Z" finished-at="2019-11-19T17:14:55Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.mumbaiMaladSignaturePckg(HomePage.java:535)at core.tests.Package.tc_Packge_MumbaiMaladSignaturePackage(Package.java:98)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_MumbaiMaladSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1544" started-at="2019-11-19T17:14:55Z" finished-at="2019-11-19T17:14:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_MumbaiMaladSignaturePackage status=FAILURE method=Package.tc_Packge_MumbaiMaladSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22953" started-at="2019-11-19T17:14:57Z" finished-at="2019-11-19T17:15:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_MumbaiVashiDeluxePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_MumbaiVashiDeluxePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_MumbaiVashiDeluxePackage" duration-ms="98448" started-at="2019-11-19T17:15:20Z" finished-at="2019-11-19T17:16:58Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.mumbaiVashiDeluxePckg(HomePage.java:592)at core.tests.Package.tc_Packge_MumbaiVashiDeluxePackage(Package.java:122)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_MumbaiVashiDeluxePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1383" started-at="2019-11-19T17:16:58Z" finished-at="2019-11-19T17:17:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_MumbaiVashiDeluxePackage status=FAILURE method=Package.tc_Packge_MumbaiVashiDeluxePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22876" started-at="2019-11-19T17:17:00Z" finished-at="2019-11-19T17:17:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_MumbaiVashiSuitePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_MumbaiVashiSuitePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_MumbaiVashiSuitePackage" duration-ms="97616" started-at="2019-11-19T17:17:23Z" finished-at="2019-11-19T17:19:00Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.mumbaiVashiSuitePckg(HomePage.java:573)at core.tests.Package.tc_Packge_MumbaiVashiSuitePackage(Package.java:114)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_MumbaiVashiSuitePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1424" started-at="2019-11-19T17:19:00Z" finished-at="2019-11-19T17:19:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_MumbaiVashiSuitePackage status=FAILURE method=Package.tc_Packge_MumbaiVashiSuitePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22612" started-at="2019-11-19T17:19:02Z" finished-at="2019-11-19T17:19:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_PuneKalyaniNagarLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_PuneKalyaniNagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_PuneKalyaniNagarLuxuryPackage" duration-ms="97700" started-at="2019-11-19T17:19:24Z" finished-at="2019-11-19T17:21:02Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.puneKalyaniNagarLuxuryPckg(HomePage.java:668)at core.tests.Package.tc_Packge_PuneKalyaniNagarLuxuryPackage(Package.java:154)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_PuneKalyaniNagarLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1484" started-at="2019-11-19T17:21:02Z" finished-at="2019-11-19T17:21:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_PuneKalyaniNagarLuxuryPackage status=FAILURE method=Package.tc_Packge_PuneKalyaniNagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22431" started-at="2019-11-19T17:21:03Z" finished-at="2019-11-19T17:21:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_PuneKalyaniNagarSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_PuneKalyaniNagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_PuneKalyaniNagarSignaturePackage" duration-ms="99396" started-at="2019-11-19T17:21:26Z" finished-at="2019-11-19T17:23:05Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.puneKalyaniNagarSignaturePckg(HomePage.java:649)at core.tests.Package.tc_Packge_PuneKalyaniNagarSignaturePackage(Package.java:146)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_PuneKalyaniNagarSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1473" started-at="2019-11-19T17:23:05Z" finished-at="2019-11-19T17:23:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_PuneKalyaniNagarSignaturePackage status=FAILURE method=Package.tc_Packge_PuneKalyaniNagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22145" started-at="2019-11-19T17:23:07Z" finished-at="2019-11-19T17:23:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_PuneShivajiNagarLuxuryPackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_PuneShivajiNagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_PuneShivajiNagarLuxuryPackage" duration-ms="99385" started-at="2019-11-19T17:23:29Z" finished-at="2019-11-19T17:25:08Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.puneShivajiNagarLuxuryPckg(HomePage.java:630)at core.tests.Package.tc_Packge_PuneShivajiNagarLuxuryPackage(Package.java:138)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_PuneShivajiNagarLuxuryPackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1414" started-at="2019-11-19T17:25:08Z" finished-at="2019-11-19T17:25:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_PuneShivajiNagarLuxuryPackage status=FAILURE method=Package.tc_Packge_PuneShivajiNagarLuxuryPackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="runOncePerMethod(org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:core.tests.Package@27efef64]" name="runOncePerMethod" is-config="true" duration-ms="22332" started-at="2019-11-19T17:25:10Z" finished-at="2019-11-19T17:25:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@60df60da]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void core.tests.Package.tc_Packge_PuneShivajiNagarSignaturePackage() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOncePerMethod -->
        <test-method status="FAIL" signature="tc_Packge_PuneShivajiNagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64]" name="tc_Packge_PuneShivajiNagarSignaturePackage" duration-ms="99238" started-at="2019-11-19T17:25:32Z" finished-at="2019-11-19T17:27:11Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failure in Creating Control => Platform: ANDROID, Page: HomePage, Control Name: payNowBtn, Locator Type: XPATH, Locator Value: //*[contains(@text, 'PAY NOW')]. Exception: Index: 0, Size: 0at control.Control.throwControlCreationException(Control.java:73)at control.Control.createControl(Control.java:101)at page.Page.getControl(Page.java:59)at core.pages.HomePage.puneShivajiNagarSignaturePckg(HomePage.java:611)at core.tests.Package.tc_Packge_PuneShivajiNagarSignaturePackage(Package.java:130)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_Packge_PuneShivajiNagarSignaturePackage -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:core.tests.Package@27efef64]" name="tearDown" is-config="true" duration-ms="1521" started-at="2019-11-19T17:27:11Z" finished-at="2019-11-19T17:27:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc_Packge_PuneShivajiNagarSignaturePackage status=FAILURE method=Package.tc_Packge_PuneShivajiNagarSignaturePackage()[pri:4, instance:core.tests.Package@27efef64] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- core.tests.Package -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
